devel/apache-ivy/Portfile

```diff
@@ -48,6 +48,6 @@ destroot {
 }
 
 platform darwin {
-    build.env       JAVA_HOME=/Library/Java/Home
+    build.env       [exec /usr/libexec/java_home]
```

Comment:
@neverpanic Did you say somewhere that we wanted to avoid `exec` because it defeats our sandboxing?